pkg/services/control/server/helpers.go

```diff
@@ -11,10 +16,18 @@ func (s *Server) getShardIDList(raw [][]byte) []*shard.ID {
 		return res
 	}
 
-	info := s.s.DumpInfo()
+	info := s.storage.DumpInfo()
 	res := make([]*shard.ID, 0, len(info.Shards))
 	for i := range info.Shards {
 		res = append(res, info.Shards[i].ID)
 	}
 	return res
 }
+
+func (s *Server) ready() error {
+	if !s.available.Load() {
+		return status.Error(codes.Unavailable, "node has not started fully yet")
```

Comment:
to me `node` is smth higher level. I'd report `service has not been completely initialized yet`

Reply:
fixed